lib/game.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'frame'
+require_relative 'score'
+
+class Game
+  attr_reader :frame, :action
+  def initialize
+    @frame_class = Frame
+    @frame = 0
+    @score = Score.new
+    @game_score = Array.new(10, 0)
+  end
+
+  def roll
+    if @frame == 9
+      roll1, roll2, roll3 = @frame_class.new.frame_10_start
+      returned = @score.calculate_frame10(roll1, roll2, roll3)
+      store_score(returned[0], returned[1], returned[2])
+    else
+      roll1, roll2, bonus = @frame_class.new.frame_start
+      returned = @score.calculate(roll1, roll2, bonus)
+      store_score(returned[0], returned[1], returned[2])
+    end
+    @frame += 1
```

Comment:
might be clearer if frame started at 1, unless it needs to start at 0 for the code